docker-compose.yaml

```diff
@@ -0,0 +1,95 @@
+version: '3'
+services:
+  hmi-client:
+    build:
+      context: ./packages/client
+      dockerfile: hmi-client/docker/Dockerfile.dev
+    environment:
+      - BYPASS_KEYCLOAK=true
+      - VITE_BYPASS_KEYCLOAK=true
+      - hmi_server_host=hmi-server
+      - hmi_server_port=3000
+    volumes:
+      - ./packages/client/hmi-client/src:/hmi-client/src
+    ports:
+      - 8080:8080
+  hmi-server:
+    build:
+      context: '.'
+      dockerfile: packages/services/hmi-server/docker/Dockerfile.dev
+    ports:
+      - 3000:3000
+    environment:
+      - QUARKUS_OIDC_ENABLED=false
+      - QUARKUS_KEYCLOAK_POLICY_ENFORCER_ENABLE=false
+      - QUARKUS_SECURITY_AUTH_ENABLED_IN_DEV_MODE=false
+      - QUARKUS_HTTP_HOST=0.0.0.0
+      - terarium-db-host=terarium-db
+      - terarium-db-port=5432
+      - data-service-host=data-service
+      - data-service-port=8000
+  terarium-db:
+    image: postgres:14.4-alpine
```

Comment:
What is `terarium-db` is it what we call `hmi-postgres`?
Also we try to use `ghcr.io/darpa-askem/postgres:15.1` to avoid issue in the long run,

Reply:
Pretty sure I called it that because that's what it's called in the config file:

https://github.com/DARPA-ASKEM/Terarium/blob/main/packages/services/hmi-server/src/main/resources/application.properties#L19

Reply:
I see, then, yes, they are the same thing!